tests/integration/Microsoft.Bot.Builder.Integration.AspNet.Core.Tests/CloudAdapterTests.cs

```diff
@@ -825,14 +825,35 @@ public async Task ExpiredTokenShouldThrowUnauthorizedAccessException()
             var httpResponseMock = new Mock<HttpResponse>().SetupAllProperties();
             httpResponseMock.Setup(r => r.Body).Returns(response);
 
+            var loggerMock = new Mock<ILogger<CloudAdapter>>();
+            var mockHttpMessageHandler = new Mock<HttpMessageHandler>();
+            mockHttpMessageHandler.Protected()
+                .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.IsAny<HttpRequestMessage>(), ItExpr.IsAny<CancellationToken>())
+                .Returns((HttpRequestMessage request, CancellationToken cancellationToken) => Task.FromResult(CreateInternalHttpResponse()));
+
+            var httpClient = new HttpClient(mockHttpMessageHandler.Object);
+
+            var httpClientFactoryMock = new Mock<IHttpClientFactory>();
+            httpClientFactoryMock.Setup(cf => cf.CreateClient(It.IsAny<string>())).Returns(httpClient);
+
             var bot = new InvokeResponseBot();
 
             // Act
-            var adapter = new CloudAdapter();
+            var cloudEnvironment = BotFrameworkAuthenticationFactory.Create(null, false, null, null, null, null, null, null, null, new PasswordServiceClientCredentialFactory(), new AuthenticationConfiguration(), httpClientFactoryMock.Object, null);
+            var adapter = new CloudAdapter(cloudEnvironment, loggerMock.Object);
```

Comment:
I think the cloudEnvironment is not necessary and all its setup, you can just do `BotFrameworkAuthenticationFactory.Create()` as in the https://github.com/microsoft/botbuilder-dotnet/blob/55496e87c0f4b1728c990653313b8c2a9624715c/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/CloudAdapter.cs#L35 line shows.

Reply:
Done! Thanks.